typescript/looker/rtl/transport.ts

```diff
@@ -217,6 +226,32 @@ export interface ITransportSettings {
   encoding?: string | null
 }
 
+/**
+ * MIME patterns for binary content types
+ * @type {RegExp}
+ */
+export const contentPatternBinary = /^application\/.*(\bjson\b|\bxml\b|\bsql\b|\bgraphql\b)|^text/i
+
+/**
+ * MIME patterns for "string" content types
+ * @type {RegExp}
+ */
+export const contentPatternString = /(^image\/|^audio\/|^video\/|^application\/.*(\bpdf\b|\bwbxml\b|\bzip\b|\bgzip\b))|charset.*=/i
+
+/**
+ * Is the content type binary or "string"?
+ * @param {string} contentType
+ * @returns {ResponseMode.binary | ResponseMode.string}
+ */
+export function responseMode(contentType: string) {
+  if (contentType.match(contentPatternBinary)) {
+    return ResponseMode.string
+  } else if (contentType.match(contentPatternString)) {
+    return ResponseMode.binary
```

Comment:
Why does `contentPatternBinary` contain the pattern for string and return string? same question in reverse for `contentPatternString`

Reply:
Whoops! Fixed and tightened up the patterns a bit.